api/src/utils/pipes/search-filter.pipe.ts

```diff
@@ -13,6 +13,7 @@ import {
   ArgumentMetadata,
   Logger,
 } from '@nestjs/common';
+import _ from 'lodash';
```

Comment:
```suggestion
import escapeRegExp from 'lodash/escapeRegExp';
```
I think this way is better

Reply:
You are right @marrouchi the recommended way will use less size :rocket: 
![image](https://github.com/user-attachments/assets/825cfdc7-fceb-4ec1-8aa1-ae680c88a833)
![image](https://github.com/user-attachments/assets/8d679ef4-edef-48dd-9fc7-882ac82830ad)